vsintegration/tests/UnitTests/LegacyLanguageService/Tests.LanguageService.Completion.fs

```diff
@@ -5319,17 +5312,6 @@ let x = query { for bbbb in abbbbc(*D0*) do
             "[<"
             ["AttributeUsage"]
             []
-      
-    [<Test>]
-    member this.``Attributes.CanSeeOpenNamespaces.Bug268290.Case2``() =
-        AssertCtrlSpaceCompleteContains 
```

Comment:
Hmm do you know why this test case was removed?

Reply:
I don't think I remember; I should try to enable it again.